tests/login.yaml

```
@@ -0,0 +1,29 @@
+appId: com.anonymous.PublicMobileApp
+---
+# ----------------------------------- LOGIN TEST -------------------------------------------------------------------
+- launchApp
+- waitForAnimationToEnd
+- assertVisible: "Login"
+
+# check fields 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)